utils.py

```diff
@@ -843,14 +927,16 @@ def get_supported_audio_language_description(language_code):
     Raises:
         Exception. If the given language code is unsupported.
     """
-    for language in constants.SUPPORTED_AUDIO_LANGUAGES:
+    for language in (
+            constants.SUPPORTED_AUDIO_LANGUAGES): # type: ignore[attr-defined]
```

Comment:
```suggestion
    for language in constants.SUPPORTED_AUDIO_LANGUAGES: # type: ignore[attr-defined]
```

Reply:
The problem here is that linter is throwing errors 'line too long' for `# type:` pragma exceeding the 80 length too. I have discussed with Sandeep and Sajal and I am working on this. I haven't found a solution for this yet. @vojtechjelinek Please let me know if you know a fix for this. Thanks!